hydroflow/examples/two_pc/main.rs

```diff
@@ -0,0 +1,70 @@
+use clap::{ArgEnum, Parser};
+use coordinator::run_coordinator;
+use hydroflow::tokio;
+use serde::Deserialize;
+use subordinate::run_subordinate;
+
+use std::error::Error;
+use std::fs::File;
+use std::io::BufReader;
+use std::path::Path;
+
+mod coordinator;
+mod protocol;
+mod subordinate;
+
+// This is a distributed version of the covid tracing app. It somewhat
+// arbitrarily splits apart two "responsibilities" of the app to exercise
+// network communication.
```

Comment:
Use `//!` (instead of) `//` for module doc comments

Reply:
Fixed. Wrong comment anyhow.